frontend/javascripts/oxalis/model/accessors/dataset_accessor.ts

```diff
@@ -345,6 +345,34 @@ export function getDatasetExtentAsString(
     formatNumberToLength(length, LongUnitToShortUnitMap[dataset.dataSource.scale.unit]),
   );
 }
+function getDatasetExtentAsProduct(extent: {
+  width: number;
+  height: number;
+  depth: number;
+}) {
+  const safeMultiply = (a: number, b: number) => {
+    const result = a * b;
+    if (!Number.isFinite(result)) {
+      throw new Error("Dataset extent product exceeds maximum safe number");
+    }
+    return result;
+  };
```

Comment:
hmmm, I'm not really fond of this. as a preface, I think, that it's unrealistic that we will ever run into this (maximum number in JS is 2^1024 which means that one dimension of a dataset would roughly have to have 2^342 voxels at least for this to be triggered). so, I wouldn't want to clutter the code here with that.

my suggestion would be: remove the safeMultiply + try-catch stuff here in this module and then simply ensure that `formatNumberToVolume` and `formatVoxels` can deal with `Infinity` as an input (not sure what it does currently).